GetStream/stream-chat-swift
# [CIS-2201] Expose channel own capabilities

### 🔗 Issue Links
CIS-2201
https://github.com/GetStream/stream-chat-swift/issues/2140

### 🎯 Goal
Expose the `channel.ownCapabalities`. This PR only parses and exposes the channel's own capabilities, it does not make use of it in the UI SDK. 

### 🧪 Manual Testing Notes
- Open a channel
- Tap on the debug button
- Tap on channel info
- Check that the channel capabilities are there

### ☑️ Contributor Checklist
